src/futures_map.rs

```diff
@@ -122,6 +122,39 @@ where
             }
         }
     }
+
+    /// Returns an iterator over all futures of type `T` pushed via [`FuturesMap::try_push`].
+    /// The order that futures are returned is not guaranteed.
```

Comment:
```suggestion
    /// The order that futures are returned in is not guaranteed.
```

I am not a native speaker but this sounds slightly better?

Reply:
Good point! Either works, but I think I'd prefer to have a positive statement, because they're easier to read:
```suggestion
    /// This iterator returns futures in an arbitrary order, which may change.
```